tests/Php74/Php74Test.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Polyfill\Tests\Php74;
+
+use ArrayObject;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @author Ion Bazan <ion.bazan@gmail.com>
+ */
+class Php74Test extends TestCase
+{
+    public function testGetMangledObjectVarsOnObject()
+    {
+        $obj = new B();
+        $obj->dyn = 5;
+        $obj->{'6'} = 6;
+
+        $this->assertSame(array(
+            "\0".'Symfony\Polyfill\Tests\Php74\B'."\0".'priv' => 4,
+            'pub' => 1,
+            "\0".'*'."\0".'prot' => 2,
+            "\0".'Symfony\Polyfill\Tests\Php74\A'."\0".'priv' => 3,
+            'dyn' => 5,
+            6 => 6,
+        ), get_mangled_object_vars($obj));
+
+        $this->assertNull(@get_mangled_object_vars(1));
```

Comment:
Can we test that an error was triggered with PHPUnit?

If its  possible it may be nice to make sure that happens as well.